tests/parsing_data/test_parse_data_from_files.py

```diff
@@ -18,6 +19,8 @@ def test_parse_dwd_data():
     file_in_bytes.seek(0)
 
     station_data = parse_dwd_data(
-        filenames_and_files=[(filename, file_in_bytes)])
+        filenames_and_files=[(filename, file_in_bytes)],
+        time_resolution=TimeResolution.DAILY
```

Comment:
Please add a test for hourly data.

Reply:
A test case serving as a "this does not raise any exceptions" has been added through a quick data collection test.

I am planning to extend the test suite significantly as soon as I will add mappings for meaningful column/field names for all resolutions available, see #54.

Are you fine with that?